### PR TITLE
Re-order contact group (#19)

### PR DESCRIPTION
**Pull request by Yiin**
demo

There is pressDelay of 100ms (for better mobile experience)

### CODE REVIEW

File: src/app/components/ContactGroupsModal.js [hunk 1]
```diff
@@ -31,6 +32,12 @@ const ContactGroupsTable = ({ onClose }) => {
     const api = useApi();
     const { call } = useEventManager();
 
+    const [list, setContactGroups] = useState(contactGroups || []);
```
Comment (EpokK) [inline src/app/components/ContactGroupsModal.js]: I prefer if you do `const [contactGroups = []] = useContactGroups`. @mmso needs to confirm.

Comment (mmso) [reply src/app/components/ContactGroupsModal.js]: You can assume that contactGroups will always be loaded at this stage so no need to do `|| []`

**Comment by EpokK** [comment]
Is it possible to allow drag and drop only from the left icon @Yiin ?

**Comment by Yiin** [comment]
@EpokK yes, of course

**Pull request merged**